risc0/circuit/rv32im/src/execute/bigint.rs

```diff
@@ -70,7 +70,7 @@ impl<Risc0ContextT: Risc0Context> BigIntIO for BigIntIOImpl<'_, Risc0ContextT> {
         check_bigint_addr(start_addr, self.mode)?;
 
         let word_count = (count + 3) / 4;
-        let mut limbs = SmallVec::<[u32; 8]>::with_capacity(word_count as usize);
+        let mut limbs = SmallVec::<[u32; 12]>::with_capacity(word_count as usize);
```

Comment:
I would leave a small comment here about where 12 came from, now that you have some actual data, less we accidentally change it later thinking it was arbitrary.

Reply:
Done!